core/src/repository/RepositoryMapper.php

```diff
@@ -2,31 +2,46 @@
 
 namespace harmony\core\repository;
 
-use harmony\core\repository\mapper\Mapper;
+use harmony\core\repository\mapper\GenericMapper;
 use harmony\core\repository\operation\Operation;
 use harmony\core\repository\query\Query;
 use harmony\core\shared\collection\GenericCollection;
 
+/**
+ * @template   Tmodel
+ * @template   Tentity
+ * @implements GetRepository<Tmodel>
+ * @implements PutRepository<Tmodel>
+ */
 class RepositoryMapper implements GetRepository, PutRepository, DeleteRepository
 {
-    /** @var GetRepository */
+    /** @var GetRepository<Tentity> */
     private $getRepository;
-    /** @var PutRepository */
+    /** @var PutRepository<Tentity> */
```

Comment:
Types should be in pascal case, right ? . So here we should use `TEntity` instead of `Tentity`

Reply:
Yep, changed.